pwa/src/templates/templateParts/footer/FooterTemplate.module.css

```diff
@@ -14,6 +14,7 @@
 
 .imagesContainer {
   display: flex;
+  justify-content: space-evenly;
```

Comment:
I think the images may be aligned left (with some margin between them).

Reply:
Ik maak 
.imagesContainer > :not(:last-child) {
  margin-right: var(--nlportal-space-inline-xl);
}